src/mame/drivers/itt3030.cpp

```diff
@@ -616,31 +527,133 @@ static GFXDECODE_START( itt3030 )
 	GFXDECODE_ENTRY( "gfx1", 0, charlayout,     0, 1 )
 GFXDECODE_END
 
+PALETTE_INIT_MEMBER(itt3030_state, itt3030)
+{
+	palette.set_pen_color(0, rgb_t::black());
+	palette.set_pen_color(1, rgb_t(215, 229, 82));
+	palette.set_pen_color(2, rgb_t::black());
+}
+
+//**************************************************************************
+//  SOUND
+//**************************************************************************
 
-void itt3030_state::machine_start()
+WRITE8_MEMBER( itt3030_state::beep_w )
 {
-	save_item(NAME(m_kbdread));
-	m_48kbank->space(AS_PROGRAM).install_ram(0, m_ram->size() - 16384, m_ram->pointer());
-	m_maincpu->space(AS_PROGRAM).install_ram(0xc000, 0xffff, m_ram->pointer() + m_ram->size() - 16384);
-	m_gfxdecode->gfx(1)->set_colorbase(1);
+	m_beep->set_state(data&1);
+}
 
-	m_kbdclk = 0;   // must be initialized here b/c mcs48_reset() causes write of 0xff to all ports
+//**************************************************************************
+//  FLOPPY
+//**************************************************************************
+
+WRITE_LINE_MEMBER(itt3030_state::fdcirq_w)
+{
+	m_fdc_irq = state;
 }
 
-void itt3030_state::machine_reset()
+#include "debugger.h"
```

Comment:
Please put this with the other `#include` directives at the top of the file - doing stuff before including a framework header can lead to chaos.